tailwind.config.ts

```diff
@@ -10,13 +9,18 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors:{
+        "btn-blue":"rgb(112, 187, 253)",
```

Comment:
ideally, we would want something generic here too. e.g. `primary-[1-9]00`, `secondary-[1-9]00`. but it's fine for the time being I guess.